liquidity-rewards/contracts/dependencies/PoolTokenWrapper.sol

```diff
@@ -12,25 +12,50 @@ contract PoolTokenWrapper {
     using SafeMath for uint256;
     using SafeERC20 for IERC20;
 
+    /**
+     * Pool token ERC20 token interface.
+     */
     IERC20 public poolToken;
 
+    /**
+     * Keep track total supply of the pool token deposited into this contract, this
+     * will reduce amount of calculations vs calling a balanceOf method.
+     */
     uint256 private _totalSupply;
+    
+    /**
+     * User balances of the pool token.
+     */
     mapping(address => uint256) private _balances;
 
+    /**
+     * Returns value of the _totalSupply variable.
+     */
     function totalSupply() public view returns (uint256) {
         return _totalSupply;
     }
```

Comment:
If we use `uint256 public _totalSupply;` instead of `uint256 private _totalSupply;`, will this whole function become unnecessary?

Reply:
Yes, we could do that in a separate commit.